closure/goog/net/xhriopool_test.js

```diff
@@ -48,6 +48,8 @@ function testSetHeadersForInitializedPoolObjects() {
              xhrIo.headers.containsKey('X-Foo'));
   assertEquals('Request should contain right header value', 'Bar',
                xhrIo.headers.get('X-Foo'));
+  assertTrue('withCredentials should be set on a request object',
```

Comment:
Instead of piggy-backing on the `testSetHeaders` method for this assertion, can you create a separate `testSetCredentials..`?